src/databricks/labs/ucx/hive_metastore/objects.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class Catalog:
```

Comment:
~this data structure seems to be used only in one place. why do we need to create whole new modules for those? also, it really is `tuple[str,str] = ("CATALOG", name)`~

might be a good idea to co-locate next to `Table` in `src/databricks/labs/ucx/hive_metastore/tables.py`. make sure to implement only the required interface methods.